homeassistant/components/vacuum/neato.py

```diff
@@ -147,17 +140,17 @@ def battery_level(self):
         return self._state['details']['charge']
 
     @property
-    def status(self):
+    def state(self):
         """Return the status of the vacuum cleaner."""
-        return self._status_state
+        return self._clean_state
 
     @property
     def state_attributes(self):
```

Comment:
Please also fix this old code and use `device_state_attributes` here instead. State attributes will be updated with `device_state_attributes` in the base entity.